Sources/SQL/SQLLiteral.swift

```diff
@@ -12,6 +12,12 @@ public protocol SQLLiteral: SQLSerializable {
     /// Creates a new `SQLLiteral` from a numeric string (no quotes).
     static func numeric(_ string: String) -> Self
     
+    /// Creates a new `SQLLiteral` from a raw string (no quotes).
+    /// Semantically different from `.numeric`:
+    /// Can be used to manually provide raw SQL code that would be difficult or
+    /// (currently) impossible to model with this library.
+    static func raw(_ string: String) -> Self
```

Comment:
We need to provide a default implementation for this method in case any packages depending on this code declare `SQLLiteral` conformers.

Simply forwarding to `.numeric` seems like a good default. (With a FIXME: to remind us to remove next release).

Reply:
Fixed. Please take another look.